modules/bbp-simulation/src/main/resources/schemas/hbphackaton/simulation/cellplacement/v0.1.0.json

```diff
@@ -0,0 +1,26 @@
+{
+  "@context": [
+    "{{base}}/contexts/nexus/core/schema/v0.1.0",
```

Comment:
there is also the nsg schema context now which would reduce the context to one line here